content/workshops/openshift_4_101/lab6-replicationrecovery.md

```diff
@@ -30,17 +30,17 @@ $ oc scale --replicas=4 dc/dc-metro-map
 $ oc get pods
 ```
 
-Notice that you now have 4 unique pods availble to inspect.  If you want go ahead and inspect them, using 'oc describe pod/<POD NAME>', you can see that each have their own IP address and logs.
+Notice that you now have 4 unique pods availble to inspect.  If you want go ahead and inspect them, using 'oc describe pod/<b>POD NAME</b>', you can see that each have their own IP address and logs.
```

Comment:
*available*
If you want to inspect your pods, use 'oc describe pod/<b>POD NAME</b>' to view the IP address and logs associated with each pod.